src/Events/Program.cs

```diff
@@ -14,13 +13,13 @@ public static void Main(string[] args)
             {
                 webBuilder.UseStartup<Startup>();
                 webBuilder.ConfigureLogging((hostingContext, logging) =>
-                    logging.AddSerilog(hostingContext, e =>
+                    logging.AddSerilog(hostingContext, (e, globalSettings) =>
                     {
                         var context = e.Properties["SourceContext"].ToString();
                         if (context.Contains("IdentityServer4.Validation.TokenValidator") ||
                             context.Contains("IdentityServer4.Validation.TokenRequestValidator"))
                         {
-                            return e.Level > LogEventLevel.Error;
+                            return e.Level >= globalSettings.MinLogLevel.EventsSettings.IdentityToken;
```

Comment:
Also changed from `Error` to `Fatal`

Reply:
These are actually consistent, original was `>` not `>=`, but I wanted consistency in _**Min**_LoggingLevel